.github/workflows/reusable_kernel_tests.yaml

```diff
@@ -8,16 +8,18 @@ on:
         type: string
         required: false
         default: master
-  push:
-    branches:
-      - master
-    tags:
-      - '[0-9]+.[0-9]+.[0-9]+\+driver'
-    
+  workflow_call:    
+    inputs:
+      libsversion:
+        description: libs version to be tested
+        type: string
+        required: false
+        default: master
+
 concurrency:
   group: kernel-tests
-  cancel-in-progress: true
-
+  cancel-in-progress: false
```

Comment:
If any other workflow is using this reusable workflow, queue the new one, without cancelling in progress workflow.
This would allow a master CI to get triggered while a driver-release is happening, and that would then wait at kernel-tests step to finish.

See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idconcurrency